src/containers/ProjectContainer/TeststepContainer/index.jsx

```diff
@@ -140,38 +154,26 @@ const TeststepContainer = (props) => {
         toggleAddTestdataForm();
     }
 
-    useEffect(() => {
-        setTestdataFormData(p => ({
-            ...p,
-            teststep: selectedItem?.id || INITIAL_TESTDATA_FORM_DATA.id,
-            payload: JSON.stringify(selectedItem?.payload?.payload || INITIAL_TESTDATA_FORM_DATA.payload),
-            parameters: selectedItem?.payload?.parameters || INITIAL_TESTDATA_FORM_DATA.parameters,
-            expected_outcome: selectedItem?.payload?.expected_outcome || INITIAL_TESTDATA_FORM_DATA.expected_outcome
-        }))
-    }, [selectedItem])
-    
-    
-
     return (
         <>
-            <SubComponentsNav 
+            <SubComponentsNav
                 title="Teststeps"
                 data={teststeps}
                 isLoading={isLoading}
                 onAddBtnClick={() => navigate(`/project/${projectName}/teststeps/new`)}
                 onSelectItemUrl={`/project/${projectName}/teststeps`}
             />
-            {cat?(
-                <AddNewTeststep 
+            {cat ? (
+                <AddNewTeststep
                     cat={cat}
                     isLoading={isLoading}
                     data={teststepFormData}
                     onchange={handleFormDataChange}
                     options={options}
                     handleSubmit={handleTeststepFormSubmit}
                 />
-            ):(
-                <TeststepViewComponent 
+            ) : (
+                <TeststepViewComponent
                     isLoading={isLoading}
                     data={selectedItem}
                     projectName={projectName}
```

Comment:
add 1 new line at ending of every file. here after line no. 196 add a new line

Reply:
done 👍🏻